src/snippets/stage-autocompleter.json

```diff
@@ -0,0 +1,422 @@
+// Generated from mongodb-ace-autocompleter@^0.4.1
```

Comment:
Do you think we want to have comments in JSON? it works but IDE marks this file red and shows `Comments are not permitted in JSON.` error. We can maybe use a `read.me` file instead of it in `snippets` folder.

Reply:
Yup, having a readme sounds like a good approach. Json isn't into comments hah. We could also rename the file to `generated-stage-autocompleter.json` to make the file name a bit more descriptive to its origin.

Reply:
I have added readme, but I didn't change the filename. I think without `generated` it looks cleaner, and except of this file we also have list of other files that are being generated (eg ones in the `out` folder) and they don't have any special prefix. Also, referring to other tools that generate files eg antlr, they also keep file names as it is when generating eg visitors or tokens.

Reply:
sounds good 👍